velox/vector/arrow/tests/ArrowBridgeSchemaTest.cpp

```diff
@@ -192,14 +192,15 @@ TEST_F(ArrowBridgeSchemaExportTest, scalar) {
   testScalarType(VARCHAR(), "u");
   testScalarType(VARBINARY(), "z");
 
+  options_.timestampTimeZone = "America/Los_Angeles";
```

Comment:
Do we always need a timestampTimeZone to be set or has a valid default value, current value is ""?

Reply:
@jinchengchenghh Removed. And use the default timestampTimeZone "UTC".